File: DailyFlash Solutions/DailyFlash Solutions/04-DailyFlash_Solutions/17_Jan_Solutions_One/Java/Pattern.java
/*
Program 4: Write a Program to Print following Pattern
2 4 6 8
2 4 6 8
2 4 6 8
2 4 6 8

*/

import java.io.*;

class Pattern{
	
	public static void main(String args[])throws IOException{
		for(int outerItr = 1 ; outerItr <= 4 ; outerItr++){
			
			for(int innerItr = 1 ; innerItr <= 4 ; innerItr++ ){
			
				System.out.printf("%d\t",innerItr*2);
			}
			System.out.printf("\n");	
		}

	}	
}	

File: DailyFlash Solutions/DailyFlash Solutions/04-DailyFlash_Solutions/17_Jan_Solutions_One/C/pattern.c
/*
Program 4: Write a Program to Print following Pattern
2 4 6 8
2 4 6 8
2 4 6 8
2 4 6 8

*/

#include<stdio.h>

#define UPPERLIMIT 4
#define LOWERLIMIT 1

void main(){
	
	for(int outerItr = LOWERLIMIT ; outerItr <= UPPERLIMIT ; outerItr++){
		
		for(int innerItr = LOWERLIMIT ; innerItr <= UPPERLIMIT ; innerItr++ ){
			
			printf("%d\t",innerItr*2);
		}
		printf("\n");	
	}	

}	
